src/components/sbb-link/sbb-link.tsx

```diff
@@ -17,43 +17,51 @@ import { i18nTargetOpensInNewWindow } from '../../global/i18n';
 export class SbbLink {
   /**
    * If set to true, the browser will
-   * show the download dialog on click.
+   * show the download dialog on click (optional).
    */
   @Prop() public download?: boolean;
 
-  /** The href value you want to link to */
-  @Prop() public hrefValue!: string;
+  /** The href value you want to link to (mandatory otherwise the link becomes a button)*/
+  @Prop() public href!: string;
```

Comment:
Should it not be optional now (if it is a button)?
```suggestion
  @Prop() public href?: string;
```